build/Packages.props

```diff
@@ -0,0 +1,61 @@
+<Project>
+
+<PropertyGroup>
+    <NuGetPackageVersion>4.8.0-preview1.5185</NuGetPackageVersion>
+
+    <!-- Prodcon-overrideable, using repo-toolset naming convention -->
+    <NuGetBuildTasksVersion>$(NuGetPackageVersion)</NuGetBuildTasksVersion>
+    <NuGetCommandsVersion>$(NuGetPackageVersion)</NuGetCommandsVersion>
+    <NuGetProtocolVersion>$(NuGetPackageVersion)</NuGetProtocolVersion>
+</PropertyGroup>
+
+  <ItemGroup>
```

Comment:
Specifying versions explicitly, not using `-Version` variables will break Orchestrated Build dependency uptake. Not sure if msbuild currently needs the uptake, but if it does you'll need to move to use `-Version` variables and make the PackageReference updates after the variables have been pulled from Orchestrated Build.

Reply:
I tried to leave anything that would be supplied externally. So `$(NuGetPackageVersion)` could still be provided same as before.